pkg/build/api/types.go

```diff
@@ -329,3 +329,8 @@ type GitInfo struct {
 	GitBuildSource    `json:",inline"`
 	GitSourceRevision `json:",inline"`
 }
+
+type BuildLog struct {
```

Comment:
Would it make sense to put a comment that said that the BuildLog object is used only for registering the endpoints?